tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/workspace/ImportLiferayWorkspaceOpMethods.java

```diff
@@ -103,6 +106,16 @@ protected IStatus run(IProgressMonitor monitor) {
 
 				};
 
+				addPortalRuntimeJob.addJobChangeListener(
```

Comment:
We shouldn't have to add this job change listener.  We should be able to tell the addPortalRuntimeJob to not run until after liferayProjectJob